src/test/resources/pages/InstructorHomeHTML.html

```diff
@@ -213,155 +237,173 @@ <h5 class="pull-left">
             <tbody>
                <tr id="session0">
                   <td>
-                     Second Feedback Session
+                     First Feedback Session
```

Comment:
I added new tests that sort the sessions. To do that, I had to modify the test data bundle (src/test/resources/data/InstructorHomePageUiTest3.json). I changed the start and end dates to make the sort test meaningful. Removing these changes to the test bundle restores the order. I think changing the start and end dates caused a change in the default order of these sessions, which is why it changed here.